dagobah/core/core.py

```diff
@@ -252,6 +256,37 @@ def add_task_to_job(self, job_or_job_name, task_command, task_name=None,
         job.add_task(task_command, task_name, **kwargs)
         job.commit()
 
+
+    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None,
+                        **kwargs):
+        """ Add a task to a job owned by the Dagobah instance. """
+
+        if isinstance(job_or_job_name, Job):
+            job = job_or_job_name
+        else:
+            job = self.get_job(job_or_job_name)
+
+        if isinstance(target_job, Job):
+            target_job = target_job
+        else:
+            target_job = self.get_job(target_job)
+
+        if not job:
+            raise DagobahError('job %s does not exist' % job_or_job_name)
+        if not target_job:
+            raise DagobahError('Target job %s does not exist' % target_job)
+
+        logger.debug('Adding JobTask with target job {0} to parent job {1}'.format(target_job.name, job.name))
+
+        if not job.state.allow_change_graph:
```

Comment:
Most of the other functions are doing this as a preliminary check before they do any of their other logic, right? I think that pattern would also work here just fine.